storage_scrubber/src/checks.rs

```diff
@@ -48,13 +48,13 @@ impl TimelineAnalysis {
 }
 
 pub(crate) async fn branch_cleanup_and_check_errors(
-    s3_client: &Client,
-    target: &RootTarget,
+    remote_client: &GenericRemoteStorage,
+    _target: &RootTarget,
```

Comment:
Hmm... Feels like it would be best to add an assertion for full s3 URLs or get rid of `_target`?

Reply:
got rid of target